core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/pythonworker/PythonWorkflowWorker.scala

```diff
@@ -33,7 +33,7 @@ object PythonWorkflowWorker {
 
 class PythonWorkflowWorker(
     actorId: ActorVirtualIdentity
-) extends WorkflowActor(logStorageType = "none", actorId) {
+) extends WorkflowActor(None, actorId) {
```

Comment:
please keep the keyword for readability, whenever the value itself is not self-explanatory.